### PR TITLE
fix unlit and depth/motion with unlit MSAA

### PR DESCRIPTION
**Pull request by sebastienlagarde**
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?

**Halo Effect**: None, Low, Medium, High?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: TestProjects/HDRP_Tests/ProjectSettings/EditorBuildSettings.asset [hunk 1]
```diff
@@ -188,7 +188,7 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/GraphicTests/Scenes/2x_Lighting/2307_ShadowVeryHigh.unity
     guid: c110cc3f909bb664b82eb554f0494597
-  - enabled: 1
+  - enabled: 0
```
Comment (anisunity) [inline TestProjects/HDRP_Tests/ProjectSettings/EditorBuildSettings.asset]: This is not related to the unlit fix, async compilation problem

Comment (sebastienlagarde) [reply TestProjects/HDRP_Tests/ProjectSettings/EditorBuildSettings.asset]: it is about 3001_DebugView.unity
